packages/augur-core/tests/test_warp_sync.py

```diff
@@ -34,11 +34,16 @@ def test_warp_sync(contractsFixture, augur, universe, reputationToken, warpSync)
     time.setTimestamp(disputeWindow.getEndTime())
 
     # Finalizing the warp sync market will award the finalizer REP based on time since it became finalizable
+    # This will also trigger a sweep of accumulated interest
     expectedFinalizationReward = warpSync.getFinalizationReward(market.address)
+    nextDisputeWindow = universe.getOrCreateNextDisputeWindow(False)
+    initialFeesBalance = cash.balanceOf(nextDisputeWindow)
     with PrintGasUsed(contractsFixture, "WS Market Finalization Cost", 0):
         with TokenDelta(reputationToken, expectedFinalizationReward, account, "REP reward not minted for finalizer"):
             assert market.finalize()
 
+    assert cash.balanceOf(nextDisputeWindow) > initialFeesBalance
```

Comment:
Is it feasible to calculate the expected interest instead of only asserting that it's more than zero?